app/utils/screenshots.py

```
@@ -1156,12 +1178,7 @@ def get_content_type(url, danger, stealth=False) -> (str, bool):
 
     lua = UA
     if stealth:
-        chrome_path = get_chrome_path()
-        cv = get_chrome_version(chrome_path)
-        lua = (
-            "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/%s.0.0.0 Safari/537.36"
-            % cv
-        )
+        lua = random_user_agent()
```

Ruff (0.11.9)

Local variable `lua` is assigned to but never used

Remove assignment to unused variable `lua`

(F841)